cliqz_env.sh

```diff
@@ -122,6 +122,9 @@ fi
 # set our own BUILD_ID in new build system, must be specified in format %Y%m%d%H%M%S
 export MOZ_BUILD_DATE=$CQZ_BUILD_ID
 
+export MOZ_SOURCE_REPO=https://github.com/cliqz-oss/browser-f
+export MOZ_SOURCE_CHANGESET=$CQZ_BUILD_ID
```

Comment:
@alver-cliqz let me know if you know a better way to add back this `MOZ_SOURCE_CHANGESET`. Do you remember having removed this `MOZ_SOURCE_REPO` variable as well at some point?

Reply:
we never use it before, as I remember

Reply:
Also, I don't want to use this flags on Win and Mac platform now (it could bring us some unwanted changes), so it they will work - let's use them only for linux build (not cross-build for Mac OS)

Reply:
In any case, it does not seem to work. The only way I could make it "work" was by injecting values directly in `mozilla-release/toolkit/mozapps/installer/upload-files.mk` (line 184 and 185). I thought that the build system would pick up the env variables to populate this `source-repo.h` file (used from `upload-files.mk` but that does not seem to be the case).

Reply:
@chrmod , can you please have a look. As I see - nothing changes in build scripts after merge with Firefox 73. Mai changes only switch from Python 2 to Python 3.

Reply:
The more likely explanation is that this is not due to any Firefox upstream change but the bump from Ubuntu 16.04 to Ubuntu 18.04. It could be that these two fields were optional when creating the RPM package but with a newer version they became mandatory.